packages/image-carousel/src/video-content.tsx

```diff
@@ -105,14 +129,16 @@ function VideoContent({
 
   return (
     <Frame size={size} height={height} frame={frame} onClick={onClick}>
+      <Poster style={{ backgroundImage: `url("${medium.sizes.large.url}")` }} />
       <Video
         ref={ref}
         src={medium.video?.large.url}
         controls={false}
         loop
         muted
         playsInline
-        poster={medium.sizes.large.url}
+        isOncePlayed
```

Comment:
isOncePlayed 가 true만 받게될거 같아요! 
```suggestion
        isOncePlayed={isOncePlayed}
```